### ISSUE TITLE
piwigo 14.0.0 breaks rv_gmaps

### ISSUE THREAD

**Issue by davemosk** [Piwigo/piwigo-gmaps#20]
Attempting to activate the plugin generates the following error messages:

[Sun Dec 24 09:39:25.680331 2023] [php7:warn] [pid 423226] [client 203.118.153.146:46874] PHP Warning:  fopen(./map.php): failed to open stream: Permission denied in /home/piwigo/plugins/rv_gmaps/maintain.inc.php on line 41, referer: [redacted]
[Sun Dec 24 09:39:25.680858 2023] [php7:warn] [pid 423226] [client 203.118.153.146:46874] PHP Warning:  fwrite() expects parameter 1 to be resource, bool given in /home/piwigo/plugins/rv_gmaps/maintain.inc.php on line 42, referer: [redacted]
[Sun Dec 24 09:39:25.680956 2023] [php7:warn] [pid 423226] [client 203.118.153.146:46874] PHP Warning:  fclose() expects parameter 1 to be resource, bool given in /home/piwigo/plugins/rv_gmaps/maintain.inc.php on line 43, referer: [redacted]

Environment:
[Piwigo](https://piwigo.org/) 14.0.0 
    Installed on 9 March 2015, 8 years 9 months 2 weeks ago
    Operating system: Linux
    PHP: 7.4.3-4ubuntu2.19 
    MySQL: 8.0.35-0ubuntu0.20.04.1 [2023-12-24 09:49:11]
    Graphics Library: ImageMagick 6.9.10-23
    Cache size 1246.79 Mo   calculated 9 months ago Refresh

Activated plugin list 7

    Admin Tools
    Exif View
    Fotorama
    LocalFiles Editor
    Media Icon
    RV Maps&Earth
    Take A Tour of Your Piwigo

While the plugin shows as activated, it is not functioning.

**Comment by ClaudiaLev** [comment]
I think Piwigo 14 is not compatible with rv_maps.
When you update to 14 it shows a message that rv_maps is not supported.

There were even problems with this plugin before with version 13: https://github.com/Piwigo/piwigo-gmaps/issues/19

**Comment by davemosk** [comment]
Well, it would be good to get these problems fixed, eh!

**Comment by ClaudiaLev** [comment]
Yes, this would be fine, but I think that this plugin is not longer under maintenance and no one feels responsible.  I switched to the Open Street Map-Plugin. It is not as good as rv_Maps, but it works.

**Comment by davemosk** [comment]
Thank you for the recommendation, @ClaudiaLev - that works well!  Much appreciated.
